helm_deploy/hmpps-dpr-tools-api/values.yaml

```diff
@@ -20,6 +20,7 @@ generic-service:
     SPRING_PROFILES_ACTIVE: "logstash"
     APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY)"
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
+    REDSHIFT_DATA_DB: datamart
```

Comment:
Would we also need DPR_LIB_AWS_STS_ENABLED set to true?

Reply:
Nope, it's enabled in the application.yml.

You gave me a "How is any of this working??" moment there. 😅

Reply:
I thought the same. I remember I deployed a version from my branch with the previous configuration (AWS clients being in the tools-api) to be able to run a demo and I wasn't sure if the version running now on Dev is from main or still my branch.